src/util/debug.ml

```diff
@@ -0,0 +1,6 @@
+open GoblintCil
+
+let f = Printf.sprintf
+let pf fmt = Printf.ksprintf print_endline fmt
+let df fmt = Pretty.gprintf (Pretty.sprint ~width:max_int) fmt
+let dpf fmt = Pretty.gprintf (fun doc -> print_endline @@ Pretty.sprint ~width:max_int doc) fmt
```

Comment:
Instead of introducing this extra mechanism, it might be nice to rely on Tracing which we already use in a lot of places throughout Goblint.

Reply:
Oops, this is still temporary debugging code.